xml/System.IO/WindowsRuntimeStorageExtensions.xml

```diff
@@ -157,10 +187,31 @@
         <Parameter Name="windowsRuntimeFile" Type="Windows.Storage.IStorageFile" RefType="this" />
       </Parameters>
       <Docs>
-        <param name="windowsRuntimeFile">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="windowsRuntimeFile">The Windows Runtime <see xref="T:Windows.Storage.IStorageFile"/> object to read from.</param>
+        <summary>Retrieves a stream for reading from a specified file.</summary>
+        <returns>A task that represents the asynchronous read operation.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+## Remarks
+
+> [!NOTE]
+> In Visual Basic and C#, you can call this method as an instance method on any object of type IStorageFile. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](~/visual-basic/programming-guide/language-features/procedures/extension-methods.md) or [Extension Methods (C# Programming Guide)](~/csharp/programming-guide/classes-and-structs/extension-methods.md).
```

Comment:
It should be ~/docs

```suggestion
> In Visual Basic and C#, you can call this method as an instance method on any object of type IStorageFile. When you use instance method syntax to call this method, omit the first parameter. For more information, see [Extension Methods (Visual Basic)](~/docs/visual-basic/programming-guide/language-features/procedures/extension-methods.md) or [Extension Methods (C# Programming Guide)](~/docs/csharp/programming-guide/classes-and-structs/extension-methods.md).
```

Reply:
I had tried that, but I was getting build warnings. OK, I'll switch it back.